packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -82,14 +88,127 @@ const List = ({
   );
 };
 
+/**
+ * Generic vertically-centered Header layout
+ */
+const Header = ({
+  flexBoxProps,
+  headingProps,
+  heading,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  heading: string;
```

Comment:
I think maybe we turn this into `string | ReactNode`, that way the user could pass something like `<>Hello <em>world</em></>` as a heading prop. What do you think about that?